simple_history/models.py

```diff
@@ -250,11 +252,24 @@ def get_prev_record(self):
         else:
             history_id_field = models.AutoField(primary_key=True)
 
+        if self.history_change_reason_field:
+            # User specific field from init
+            history_change_reason_field = self.history_change_reason_field
+        elif getattr(settings, 'SIMPLE_HISTORY_HISTORY_CHANGE_REASON_FIELD', False):
+            # User specific field from settings
+            history_change_reason_field = getattr(settings, 'SIMPLE_HISTORY_HISTORY_CHANGE_REASON_FIELD')
+            max_length = getattr(settings, 'SIMPLE_HISTORY_HISTORY_CHANGE_REASON_FIELD', 100)
+            history_change_reason_field.max_length = max_length
```

Comment:
Actually you're right @krisneuharth . This should be nullable by default and if someone passes in a change reason field, they can make that not nullable if they'd like.